osu.Game/Overlays/OverlayRulesetSelector.cs

```diff
@@ -14,6 +14,7 @@ namespace osu.Game.Overlays
     public partial class OverlayRulesetSelector : RulesetSelector
     {
         public OverlayRulesetSelector()
+            : base(true)
```

Comment:
Bool parameters as such are frowned upon, they're unparseable in non-IDE view. At least a `CanAddRuleset` method at base class that's overriden here would read much better (along with an inline comment explaining that web cannot handle custom rulesets yet).

Reply:
Init-only property is another alternative.